ilagan_andreijireh/script.js

```diff
@@ -91,4 +91,37 @@ function loadComments() {
     comments = JSON.parse(storedComments);
     displayComments();
   }
-}
\ No newline at end of file
+}
+
+document.getElementById('search-button').addEventListener('click', function() {
```

Comment:
### IDs should be written in snake case. All lower case separated by underscore.

Of course, when you commit changes to your html, you should edit your js file too for it to get the appropriate id from your HTML file.

This also applies in lines 97, 101, and 117.